README.md

```diff
@@ -1,11 +1,12 @@
 # Introduction
-Get Pushover Notifications when Services are not Running
+The SwarmAlert app monitors the availability of the services running in a Docker Swarm. If a service has no running task, the app generates a Pushover Notification.
+
 Note: This project is based on the [monitor-docker-slack](https://github.com/DennyZhang/monitor-docker-slack)
 
 # General Idea
-1. Start a service in the swarm.
-2. This service will query status for all services.
-3. Send pushover notifications.
+1. Start one or more services in the swarm.
+2. This service will monitor the availability of all services (except those in an optional whitelist).
```

Comment:
Is all services included on the whitelist,if the list is empty (or not provided) it does not monitorize anything